noxfile.py

```diff
@@ -719,16 +733,33 @@ def notebook(session):
     pytest_command = [
         "py.test",
         "--nbmake",
-        "--nbmake-timeout=600",
+        "--nbmake-timeout=900",  # 15 minutes
```

Comment:
Is there a reason why we increase the timeout limit specifically to 15 mins. Do we test the boundary?

Reply:
The default was too slow for the LLM sample when I tested locally.